fidesctl/src/fidesctl/cli/util_comands.py

```diff
@@ -121,9 +121,13 @@ def reset_db(ctx: click.Context, yes: bool) -> None:
 def view_config(ctx: click.Context) -> None:
     """
     Prints the current fidesctl configuration values.
+
+    This will print either all of the default values _or_
+    whatever values were set by the user.
```

Comment:
What problem does this solve? It seems confusing to output the config differently based on the way it's set. I'd rather see the command take an option to print only user-defined config values (and have it print everything by default).

Reply:
as a user, if i print the config I don't want to get a ton of noise, I just want to see what values I set and am using. If i didn't set the values, they don't actually do anything, right? The default values aren't really usable outside of our repo/project

Reply:
> I don't want to get a ton of noise, I just want to see what values I set and am using

IMO, "what values I set" and "what values I am using" are different things, and we should enable _both_ (while always making it clear which version is being produced). In the context you described, your change makes sense, but that context won't always be present. As a standalone command, I think the most intuitive usage is to print all current config (including default values) by default, and to enable an option that limits the output to just the user-specified options/values.

> If i didn't set the values, they don't actually do anything, right? The default values aren't really usable outside of our repo/project

The default values would be used, which _do_ do something! Whether or not that something leads to a broken implementation is for the user to decide, and exposing the default values here can be a valuable tool for debugging.

Reply:
updated